birdhouse/optional-components/testthredds/docker-compose-extra.yml

```diff
@@ -0,0 +1,42 @@
+version: '2.1'
+services:
+  proxy:
+    volumes:
+    - ./optional-components/testthredds/conf.extra-service.d:/etc/nginx/conf.extra-service.d/testthredds:ro
+    links:
+    - testthredds
+
+  testthredds:
+    image: unidata/thredds-docker:4.6.14
+    container_name: testthredds
+    ports:
+      - "8084:8080"
+    env_file:
+      - ./config/thredds/thredds.env
+    environment:
+      # for reconstructing proper URL back to user when Thredds behind proxy
+      # because Twitcher eats the "Host" http header set by Nginx
+      PAVICS_FQDN_PUBLIC: $PAVICS_FQDN_PUBLIC
+    volumes:
+      - testthredds_persistence:/usr/local/tomcat/content/thredds
+      - /data/testdatasets:/pavics-testdata:ro
+      - /data/datasets:/pavics-data:ro
+      - /data/ncml:/pavics-ncml:ro
+      - wps_outputs:/pavics-data/wps_outputs:ro
```

Comment:
Please add a wps output directory (rw) that we can use to test outputting files to this threddsserver. 

The plan will be to configure Emu to write netCDF/ncML files to this test server and then check we can access them through the opendap link sent back in the process response.

Reply:
Done, see commit comment in https://github.com/bird-house/birdhouse-deploy/pull/38/commits/c0c277cb519ad61fd4608769e9ed12eb720e9766